server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -106,14 +106,19 @@ export class ScriptoriumLambda implements IPartitionLambda {
             ...message,
             mongoTimestamp: new Date(message.operation.timestamp),
         }));
-        return runWithRetry(
-            async () => this.opCollection.insertMany(dbOps, false),
-            "insertOpScriptorium",
-            3 /* maxRetries */,
-            1000 /* retryAfterMs */,
-            getLumberBaseProperties(this.documentId, this.tenantId),
-            (error) => error.code === 11000,
-            (error) => !this.clientFacadeRetryEnabled, /* shouldRetry */
-        );
+
+        const documentId = messages.length > 0 ? messages[0].documentId : "";
+        const tenantId = messages.length > 0 ? messages[0].tenantId : "";
```

Comment:
Now that we are computing these values here based on the messages, perhaps we can remove the class members/properties `tenantId` and `documentId`?